docs/WebRTC_support_in_Clearwater.md

```diff
@@ -77,20 +74,13 @@ Using STUN and TURN
 
 **Note:** The above uses a default (non-Clearwater) STUN server, and no
 TURN server. Clearwater has its own STUN and TURN servers which can be
-used to support clients behind NATs and firewalls, but at the time of
-writing (April 2013) there are some difficulties with the WebRTC spec
-that make the Clearwater TURN server inaccessible to compliant browsers.
-These are being tracked.
+used to support clients behind NATs and firewalls.
 
 If one or both clients are behind a symmetric firewall, you must use
 TURN. Configure this as follows:
 
 -   On the 'expert mode' tab, fill in the following field with your
-    SIP username and the TURN workaround password set by your administrator
-    (`turn_workaround` in [Installing a Chef workstation](Installing_a_Chef_workstation.md)).
-    This is a workaround for Chrome WebRTC [issue
-    1508](https://code.google.com/p/webrtc/issues/detail?id=1508) which
-    will be fixed in Chrome 28 (not yet released).
+    SIP username and password.
     -   ICE Servers: [{url:"turn:&lt;phone
```

Comment:
Would it be worth mentioning here as well that WebSocket Server URL should be 'ws://<domain>:5062'